src/protect-camera.ts

```diff
@@ -51,6 +52,16 @@ export class ProtectCamera extends ProtectDevice {
     void this.configureDevice();
   }
 
+  public get cropOptions(): CropOptions {
+    return {
+      enabled: this.hasFeature("Video.Crop"),
```

Comment:
Please change the test for crop options to use the hints object. See `configureHints()` for more. The actual options themselves are fine to return via the getter…but whether the option is enabled or not belongs in hints.

Reply:
I didn't read this properly and just put everything in the hints.

Would it be better to put the crop values back in a getter?

Reply:
Okay, fixed this.